fixtures/artifactor_plugin.py

```diff
@@ -92,7 +92,7 @@ def pytest_addoption(parser):
                      help="A run id to assist in logging")
 
 
-@pytest.mark.hookwrapper
+@pytest.mark.tryfirst
 def pytest_configure(config):
     global proc
     if not SLAVEID and not proc and isinstance(art_client, ArtifactorClient):
```

Comment:
@RonnyPfannschmidt So I figured it out, there is a ```yield``` that needs to be removed below to make this work.

However then it crashes a little further on......I get this with pytest 2.9 also.

If I try to ```from fixtures.pytest_store import store``` I get an error like the following

Traceback (most recent call last):
  File "/home/cfme_tests/utils/artifactor_start.py", line 3, in <module>
    from artifactor import Artifactor, initialize
  File "/home/cfme_tests/artifactor/__init__.py", line 134, in <module>
    from utils.net import random_port
  File "/home/cfme_tests/utils/net.py", line 5, in <module>
    from fixtures.pytest_store import store
  File "/home/cfme_tests/fixtures/pytest_store.py", line 25, in <module>
    from _pytest.terminal import TerminalReporter
  File "/home/default/lib/python2.7/site-packages/_pytest/terminal.py", line 5, in <module>
    from _pytest.main import EXIT_OK, EXIT_TESTSFAILED, EXIT_INTERRUPTED, \
  File "/home/default/lib/python2.7/site-packages/_pytest/main.py", line 5, in <module>
    import pytest, _pytest
  File "/home/default/lib/python2.7/site-packages/pytest.py", line 27, in <module>
    _preloadplugins() # to populate pytest.* namespace so help(pytest) works
  File "/home/default/lib/python2.7/site-packages/_pytest/config.py", line 75, in _preloadplugins
    _preinit.append(get_config())
  File "/home/default/lib/python2.7/site-packages/_pytest/config.py", line 84, in get_config
    pluginmanager.import_plugin(spec)
  File "/home/default/lib/python2.7/site-packages/_pytest/config.py", line 384, in import_plugin
    __import__(importspec)
  File "/home/default/lib/python2.7/site-packages/_pytest/python.py", line 400, in <module>
    class PyCollector(PyobjMixin, pytest.Collector):
AttributeError: 'module' object has no attribute 'Collector'